code/main/edge/stateStoreManager.brs

```diff
@@ -0,0 +1,44 @@
+' ********************** Copyright 2024 Adobe. All rights reserved. **********************
+' *
+' * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+' * you may not use this file except in compliance with the License. You may obtain a copy
+' * of the License at http://www.apache.org/licenses/LICENSE-2.0
+' *
+' * Unless required by applicable law or agreed to in writing, software distributed under
+' * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' * OF ANY KIND, either express or implied. See the License for the specific language
+' * governing permissions and limitations under the License.
+' *
+' *****************************************************************************************
+
+function _adb_StateStoreManager() as object
+
+    return {
+        _state: invalid,
+
+        getStateStore: function() as object
+            return m._state
+        end function,
+
+        setStateStore: function(state as object) as void
+            if _adb_isEmptyOrInvalidArray(state)
+                _adb_logDebug("_adb_StateStoreManager::setStateStore() - stateStore is empty or invalid.")
+                return
+            end if
+
+            m._state = state
```

Comment:
When setting or retrieving the state store data, do you need to consider the maxAge field which defines how long the element should be stored?
https://github.com/adobe/aepsdk-edge-ios/blob/dev/Sources/EdgeNetworkHandlers/StoreResponsePayload.swift#L27

Reply:
I will update it to match the mobile SDK, in case we decide to persist these. I should have the changes in by today mostly.

Reply:
@kevinlind as per our discussion I will take this as a separate issu/PR #178